onlineweb4/urls.py

```diff
@@ -137,6 +137,9 @@
     urlpatterns += [
         url(r"^events/", include("apps.events.urls")),
         url(r"^dashboard/events/", include("apps.events.dashboard.urls")),
+        url(
+            r"^events-api/", include("apps.events.api.urls")
+        ),  # url is a dummy, but needed to import the file
```

Comment:
What does a dummy url imply? It doesn't go anywhere?

Reply:
The URL passed here is used for urls registered with django url patterns, but the file we are referencing doesn't have any django URL patterns, we just use the DRF router. So it won't be used for any urls, it is a required parameter